pkg/controller/chaosengine/chaosengine_controller.go

```diff
@@ -51,7 +51,10 @@ import (
 	"github.com/litmuschaos/chaos-operator/pkg/controller/watcher"
 )
 
-var _ reconcile.Reconciler = &ReconcileChaosEngine{}
+var (
+	finalizer                      = "chaosengine.litmuschaos.io/finalizer"
```

Comment:
It's not variable, It should be constant (never change)

Reply:
ok got it. Let me make the change.

Reply:
cool